src/component/admin/groups/GroupsList/GroupCard/GroupCard.tsx

```diff
@@ -92,13 +110,27 @@ export const GroupCard = ({ group }: IGroupCardProps) => {
                             <ConditionallyRender
                                 condition={group.projects.length > 0}
                                 show={group.projects.map(project => (
-                                    <Badge
-                                        color="secondary"
-                                        icon={<TopicOutlinedIcon />}
-                                        sx={{ marginRight: 0.5 }}
+                                    <Tooltip
+                                        key={group.id}
+                                        title="View project"
+                                        arrow
+                                        placement="bottom-end"
+                                        describeChild
                                     >
-                                        {project}
-                                    </Badge>
+                                        <Badge
+                                            onClick={e => {
+                                                e.preventDefault();
+                                                navigate(
+                                                    `/projects/${project}/access`
+                                                );
+                                            }}
+                                            color="secondary"
+                                            icon={<TopicOutlinedIcon />}
+                                            sx={{ marginRight: 0.5 }}
```

Comment:
Might be worth creating a `StyledBadge` instead of using `sx`: https://mui.com/system/basics/#performance-tradeoff

Reply:
Done